aws/resource_aws_elasticache_subnet_group.go

```diff
@@ -173,4 +173,10 @@ func resourceAwsElasticacheSubnetGroupDelete(d *schema.ResourceData, meta interf
 		}
 		return nil
 	})
+	if isResourceTimeoutError(err) {
+		_, err = conn.DeleteCacheSubnetGroup(&elasticache.DeleteCacheSubnetGroupInput{
+			CacheSubnetGroupName: aws.String(d.Id()),
+		})
+	}
+	return err
```

Comment:
Same here regarding ignoring the missing resource error and returning context for the error:

```suggestion

	if isAWSErr(err, elasticache.ErrCodeCacheSubnetGroupNotFoundFault, "") {
		return nil
	}

	if err != nil {
		return fmt.Errorf("error deleting Elasticache Subnet Group (%s): %s", d.Id(), err)
	}

	return nil
```